### ISSUE TITLE
Nested Menu & Menu items

### ISSUE THREAD

**Issue by mimarz** [equinor/design-system#2103]
When nesting `Menu`'s the subsequent `Menu.Item` with an icon are misaligned to the parent. 

Try to see if there is a way to fix this so that they are aligned.

Example with a workaround for aligned menu item with icon: https://codesandbox.io/s/muddy-violet-5mim7t?file=/src/TopbarAccessibleMenu.tsx

**Comment by mimarz** [comment]
> Should this ticket maybe be about adding support for nested menus in general? Its not like keyboard navigation works either since up/down arrow also opens the menu so you get trapped. Needs to be able to support multiple submenus i suppose?

Thats correct! Kb navigation might be finecky to sort out, but if you want to have a go at it go ahead @oddvernes

**Comment by oddvernes** [comment]
I think I will park this for now, further discussion is needed about how a possible support for nesting should look like beyond making it work with alignment and keyboards support. 
(a menu requiring nesting is probably pretty large, otherwise we have sections. So then maybe there needs to be a data structure, internal handling of state, design, multiple levels and if so how to make sure submenus opens inside viewport etc)

**Comment by akvam** [comment]
Moved to TO DO since upgrade to Floating UI is already done and collection state management is not relevant any more in Menu.

**Comment by dianacavalcanti** [comment]
Hello EDS team,

We need to create a menu with submenus, I’ve check the menu/menu.item available in EDS looks like don’t  support submenu.
We currently need too level of next menu item.
Menu
MenuItem
  MenuItem
     MenuItem
     
https://www.figma.com/design/ywFPniJaJk66PhvgFybyE4/RIPS-Portal-UI?node-id=2827-20590&t=oCsGbHSNApQn70GB-0

Any update for this feature be available in EDS? :)
